docs/user-guide/install.md

```diff
@@ -6,18 +6,73 @@ vSphere Docker Volume Service is simple to install. It has zero configuration an
 
 In addition the ```make build-all``` will generate the packages.
 
-# Install on ESX
+# Install on ESX [![VIB_Download](https://api.bintray.com/packages/vmware/vDVS/VIB/images/download.svg)](https://bintray.com/vmware/vDVS/VIB/_latestVersion)
 
 [VIB](http://blogs.vmware.com/vsphere/2011/09/whats-in-a-vib.html) and [Offline Depot](https://pubs.vmware.com/vsphere-60/index.jsp#com.vmware.vsphere.install.doc/GUID-29491174-238E-4708-A78F-8FE95156D6A3.html?resultof=%2522%256f%2566%2566%256c%2569%256e%2565%2522%2520%2522%256f%2566%2566%256c%2569%256e%2522%2520%2522%2564%2565%2570%256f%2574%2522%2520) are the packages built to install the backend for the service on ESX. The backend can be installed using esxcli or vmware tools such as [VUM](http://pubs.vmware.com/vsphere-60/topic/com.vmware.ICbase/PDF/vsphere-update-manager-601-install-administration-guide.pdf)
 
-Here is a demo show casing esxcli
+Log into ESXi host and download the [latest release](https://bintray.com/vmware/vDVS/VIB/_latestVersion) of vDVS driver VIB on ESXi and initiate the install by specifying the full path to the VIB.
 
-<script type="text/javascript" src="https://asciinema.org/a/80405.js" id="asciicast-80405" async></script>
+```
+# esxcli software vib install -v /tmp/vmware-esx-vmdkops-0.12.ccfc38f.vib
+Installation Result
+   Message: Operation finished successfully.
+   Reboot Required: false
+   VIBs Installed: VMWare_bootbank_esx-vmdkops-service_0.12.ccfc38f-0.0.1
+   VIBs Removed:
+   VIBs Skipped:
+```
```

Comment:
Good point! Instead of creating new section, shouldn't upgrade instruction goes to separate user guide. This is just an installation guide.

What do you think?